src/code-components/FileInput/FileInput.tsx

```diff
@@ -33,7 +33,8 @@ export function FileInput({
 
     if (valid.length) onChange?.(valid);
     if (invalid.length) onInvalidFileInput?.(invalid);
-    return;
+
+    inputRef.current!.value = "";
```

Comment:
This could cause runtime error if `inputRef.current` is undefined or null, why don't we use optional chaining instead of non-null assertion?

```suggestion
    inputRef.current?.value = "";
```

Reply:
I think it is unlikely that the reference would be null when calling the function from the button (using that reference), but I can check it just to be sure